portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html, body {
+  height: 100%; 
 }
 
-#greeting-container {
-  margin-top: 20px;
+.wrapper {  
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 800px;
 }
+
+.welcome {
+  height: 100%;
+}
+
+.left {
+  width: 50%;
+}
+
+.right {
+  align-self: auto;
+  text-align: center;
+  width: 50%;
+}
+
+img {
```

Comment:
Careful, these styles will be applied to all `img` elements on the page. You probably want to add _specificity_ to the selector by referring to the image as a child to some other element (ex. `.wrapper .left img {...}`)